gcf_data_mapper/parsers/family.py

```diff
@@ -139,18 +134,18 @@ def map_family_metadata(row: pd.Series) -> Optional[dict]:
         return None
 
     metadata = {
-        "approved_ref": [row.at[FamilyColumnsNames.APPROVED_REF.value]],
+        "approved_ref": [str(row.at[FamilyColumnsNames.APPROVED_REF.value])],
```

Comment:
Great - we only need to do the cast to str here - not internally - just a minor point